pangolp/mod-recruit-friend
# feat(command): Add recruit view

## Changes Proposed:
- A new command is added, which allows to see who are the recruited characters, and in this way, to obtain more information about the recruitment.
- Folder sql/delete, where you can find some of the necessary commands to delete the tables, in case the module is deleted.
- New photos and information are added in the README.md files.

## Issues Addressed:

- Closes https://github.com/pangolp/mod-recruit-friend/issues/2

## SOURCE:


## Tests Performed:

- In-game
- Windows 10


## How to Test the Changes:


1. Ingame, try the `.recruit view` command in case you have a recruited person, it should show you the names of the characters that were recruited. Otherwise, it should not show you anything.
2. In github, you can see more information about what the module offers and what it doesn't offer. Given that at no time, it was mentioned that the module delivers prizes or anything, it only serves to link the accounts, and that the experience of leveeo, is not so frustrating.
